packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -25,6 +25,11 @@ Parameters:
       S3 key, or the ID/ARN of an existing key.
     Default: ''
     Type: String
+  ImageAssetsEcrKmsKeyId:
+    Description: Empty to create a new key, 'AWS_DEFAULT_KEY' to use default
```

Comment:
I think this needs to be changed around. The default `''` should do nothing. We should introduce another option for creating a new key. Since it's not actually creating a new key, we should also explain that.

Maybe

- `''` - do nothing
- `AWS_DEFAULT_KEY` - uses default encryption
- `CREATE_NEW_CMK`? - creates new key (shares key with bucket key)

Reply:
Yeah I am not happy with the wording and the complexity that comes with it. Makes it really hard to understand what is actually happening.
What do you think about the revised names? They hopefully make it clearer what is happening.